src/it/resources/org/checkstyle/suppressionxpathfilter/matchxpath/SuppressionXpathRegressionMatchXpathEncodedAmpChar.java

```diff
@@ -0,0 +1,25 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
please remove header from all test Inputs files, such files should contain only content that is matter for test.

Reply:
Done